type-tests/promise-events.d.ts

```diff
@@ -0,0 +1,4 @@
+export class EventEmitter {
```

Comment:
Why are we defining this type here? I don't see it used either in any of the tests *or* elsewhere in the codebase. 🤔